docs/en/ingest-management/troubleshooting.asciidoc

```diff
@@ -4,72 +4,186 @@
+
+**Common problems:**
+
+* <<ingest-manager-not-in-kibana>>
+* <<ingest-management-setup-fails>>
+* <<ingest-manager-app-crashes>>
+* <<agent-enrollment-timeout>>
+
+**Frequently asked questions:**
+
+* <<enrolled-agent-not-showing-up>>
+* <<where-are-the-agent-logs>>
+* <<what-is-my-agent-config>>
+* <<where-is-the-data-agent-is-sending>>
+* <<i-deleted-my-agent>>
+* <<i-need-to-stop-agent>>
+* <<i-rebooted-my-host>>
+* <<what-is-the-endpoint-package>>
 
 [float]
-[[ingest-manager-not-in-cloud-kibana]]
-== The {ingest-manager} app is not listed in my {kib} cloud navigation list
+[[ingest-manager-not-in-kibana]]
+== The {ingest-manager} app is not listed in the {kib} side navigation
+
+In 7.8, the {ingest-manager} app is experimental. You must enable the app to
+see it in {kib}.
+
+//REVIEWERS: I don't want to have separate sections for cloud vs self-managed
+//because we are working on a widget for switching between instructions for
+//the two. Also I think it's more confusing to users if we have two sections
+//that cover the same content. 
+
+//TODO: Add platform tabs when the tabbed panel widget is stable (possibly after
+// 7.8)
+
+**To enable {ingest-manager} on cloud:**
+
+. Go to your deployment in the user console.
+
+. Under the deployment name in the side navigation, click **Edit**.
+
+. In the {kib} section, expand **User setting overrides** and enter the
+following setting:
++
+[source,yaml]
+----
+xpack.ingestManager.enabled: true
+----
+
+. Click **Save**.
+
+{kib} will restart automatically. When {kib} is available, refresh the browser
+to see the {ingest-manager} app in the navigation menu.
+
+**To enable {ingest-manager} on a self-managed cluster:**
+
+//REVIEWERS: Can't on-premise also mean cloud? I've seen self-managed used, too
+//so I've used that here. 
 
-In 7.8, the {ingest-manager} app is in Alpha state and as such is only shown when specifically enabled in the start up or cloud 'override' options. To enable {ingest-manager} on cloud, go to your deployment in the user console. Click on the 'edit' link directly beneath the deployment name. Then click on 'user setting overrides' underneath the {kib} heading. Here you must enter the following x-pack option and click the save button at the bottom:
+. In the {es} configuration file, `config/elasticsearch.yml`, set the following
+security settings to enable security and API keys:
++
+[source,yaml]
+----
+xpack.security.enabled: true
+xpack.security.authc.api_key.enabled: true
+----
 
+. In the {kib} configuration file, `config/kibana.yml`, enable {ingest-manager}
+and specify user credentials:
++
 [source,yaml]
 ----
 xpack.ingestManager.enabled: true
+xpack.ingestManager.fleet.tlsCheckDisabled: true <1>
+xpack.security.enabled: true
+elasticsearch.username: "elastic" <2>
+elasticsearch.password: "abc123iUnbRftkABC123"
 ----
+<1> This setting is not required if you configure TLS checking.
+<2> Specify a user who is authorized to use {ingest-manager} 
```

Comment:
specifically the user needs to have 'superuser' privilege.  which is a little hectic, and only for the Alpha release (we hope).

Reply:
Yah, I was conflicted about whether to cover it here because we mention that elsewhere.